bin/generate_unfiltered_sce.R

```diff
@@ -51,6 +57,16 @@ if(!(stringr::str_ends(opt$unfiltered_file, ".rds"))){
   stop("unfiltered file name must end in .rds")
 }
 
+# check that mitochondrial gene list exists
+if(!file.exists(opt$mito_file)){
+  stop("Mitochondrial gene list is required using -m or --mito_file")
```

Comment:
If the mito file  is required by option parser, then we will want to change this text.
```suggestion
  stop("Mitochondrial gene list file not found.")
```